checks.d/teamcity.py

```diff
@@ -21,7 +21,7 @@ def __init__(self, name, init_config, agentConfig, instances=None):
         # Keep track of last build IDs per instance
         self.last_build_ids = {}
 
-    def _initialize_if_required(self, instance_name, server, build_conf):
+    def _initialize_if_required(self, instance_name, server, build_conf, ssl_validation):
```

Comment:
@jslatts The window for the 5.7  release will be closing soon. Do you have a moment to look at @remh 's feedback?